bench/bench.ml

```diff
@@ -316,6 +316,42 @@ module Index = struct
         dependency = Some "replace_random";
       };
     ]
+
+  let minimal_benchmarks =
+    [
+      {
+        name = "replace_random";
+        synopsis = "Replace in random order";
+        readonly = false;
+        fresh = true;
+        benchmark = write_random;
+        dependency = None;
+      };
+      {
+        name = "replace_random_sync";
+        synopsis = "Replace in random order with sync";
+        readonly = false;
+        fresh = true;
+        benchmark = write_sync;
+        dependency = None;
+      };
+      {
+        name = "find_random_ro";
+        synopsis = "[RO] Find in random order";
+        readonly = true;
+        fresh = false;
+        benchmark = find_random;
+        dependency = Some "replace_random";
+      };
+      {
+        name = "find_random_rw";
+        synopsis = "[RW] Find in random order";
+        readonly = false;
+        fresh = false;
+        benchmark = find_random;
+        dependency = Some "replace_random";
+      };
+    ]
```

Comment:
Quite a lot of duplication of metadata here. How about adding a record field ``speed = [ `Quick | `Slow ]`` and then filtering to pick only the `` `Quick`` tests in the case where `--minimal` is passed?

Reply:
That is a really good suggestion! Thanks!